docs/reference/redirects.asciidoc

```diff
@@ -637,7 +637,7 @@ See <<ml-get-filter>> and
 [role="exclude",id="ml-event-resource"]
 === Scheduled event resources
 
-See <<ml-get-calendar-event>> and 
+See <<ml-get-calendar-event>> and
```

Comment:
I think your atom setup is removing trailing whitespace.

Not a huge issue, but it can introduce some noise in PRs and diffs.